awswrangler/s3/_read_deltalake.py

```diff
@@ -0,0 +1,159 @@
+"""Amazon S3 Read Delta Lake Module (PRIVATE)."""
+
+import logging
+from typing import Any, Dict, List, Optional, Tuple
+
+import boto3
+import pandas as pd
+from deltalake import DataCatalog, DeltaTable
```

Comment:
Deltalake is declared as optional dependency so you might want to look into dynamically loading the module. An example can be found [here](https://github.com/aws/aws-sdk-pandas/blob/main/awswrangler/oracle.py#L20).

Reply:
Oh, yes, thanks!